index.html

```diff
@@ -10,10 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    
+    <main>
```

Comment:
There's usually only one `<main>` element on the page, used for the primary content area. There are several other semantic elements you could use to "section" your content. [This part of the week 1 lesson](https://syllabus.codeyourfuture.io/html-css/week-1/lesson#sectioning-content) should have everything you need :rocket: